UPGRADE.md

```diff
@@ -80,7 +80,7 @@ With the release of Kuma 2.6.0, we've made some changes to the implementation of
 
 To ensure a smooth transition to Kuma 2.6.0, carefully review your existing configuration files and make necessary adjustments related to denied request responses and RBAC-related Envoy stats.
 
-### Make format SI valid for bandwidth in MeshFaultInjection policy
+### Make SI format valid for bandwidth in MeshFaultInjection policy
```

Comment:
this is not targetting 2.6 and it's in `UPGRADE.md` https://github.com/kumahq/kuma/blob/master/UPGRADE.md#make-format-si-valid-for-bandwidth-in-meshfaultinjection-policy - or am I confused about something?